### PR TITLE
Mon 34072 daterange conf

### PR DESCRIPTION
**Pull request by bouda1**
## Description

Attempt to add config objects for daterange, timerange.

REFS: MON-34072

### CODE REVIEW

File: engine/inc/com/centreon/engine/timeperiod.hh [hunk 1]
```diff
@@ -79,8 +79,4 @@ void get_next_valid_time(time_t pref_time,
                          time_t* valid_time,
                          com::centreon::engine::timeperiod* tperiod);
```
Comment (jean-christophe81) [inline engine/inc/com/centreon/engine/timeperiod.hh:81]: why remove it, it may be usefull to debug in future

File: engine/src/timeperiod.cc [hunk 2]
```diff
@@ -109,41 +109,6 @@ bool timeperiod::operator!=(timeperiod const& obj) noexcept {
   return !(*this == obj);
 }
```
Comment (jean-christophe81) [inline engine/src/timeperiod.cc:110]: why remove it, it may be usefull to debug in future

Comment (bouda1) [reply engine/src/timeperiod.cc:110]: I prefer to rewrite them when needed.

Comment (bouda1) [reply engine/src/timeperiod.cc:110]: I just moved these two operators to configuration.

**Pull request merged**